tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -272,6 +272,12 @@ namespace.ceph:
         used:
           help: used size
           type: int
+        create_pool:
+          help: To check if new pool has to created or not for rbd creation
+          type: bool
+        pool_parameters:
```

Comment:
I would rather like this o be called `backing_pool_details` and if this parameter is passed, a new pool would be created and rbd would get created using this new pool. With this actually we wont need `create_pool` flag as such.